reconcile/acs_policies.py

```diff
@@ -83,7 +83,7 @@ def _build_policy(
                 [
                     Scope(cluster=cluster_name_to_id[cs.name], namespace="")
                     for cs in cast(
-                        gql_acs_policies.AcsPolicyScopeClusterV1,
+                        "gql_acs_policies.AcsPolicyScopeClusterV1",
```

Comment:
What's the rationale for this? Avoiding more symbols to be imported only for type checking?

Reply:
It comes from https://docs.astral.sh/ruff/rules/runtime-cast-value/ and it's related to the `typing.TYPE_CHECKING` blocks.